script/back-end/server.go

```diff
@@ -1,11 +1,13 @@
 package main
 
 import (
+	"fmt"
     "net/http"
 )
 
 func main() {
     http.Handle("/", http.FileServer(http.Dir("./public")))
+	fmt.Println("Listening...")
     http.ListenAndServe(":8090", nil)
```

Comment:
I can do that, but the cost would be either an added dependency or additional complexity for a part of this service that I'd rather not expand beyond a basic stub. This server only serves static files, and it's really only supposed to be used during development for testing and if the app is containerized. My configuration for hosting this currently uses Amplify, which doesn't even see this file, for example.

Perhaps a good compromise would be to use JSON as a configuration file, and load it in using the relatively simple standard JSON library. Let me know if you think this would be worth the effort required, and I'll make it happen.

Reply:
I leave that decision up to you.  It makes sense that it is stored in a JSON file but that's just me.

Reply:
I'll leave it as-is for now, with the caveat that if I need to add *any* additional functionality to the stub server, I'll set up a proper config file first.